File: Data Structures & Algorithms/missing-number/submission-4.java
class Solution {
    public int missingNumber(int[] nums) {
        // [1,2,3]
        // max = nums.length - 1

        Arrays.sort(nums);

        for(int i = 0; i < nums.length; i++){
            if(i == nums[i]){
               continue;
            } else{
                return i;
            }
        }

        return nums.length;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-4.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        if nums == []:
            return 0
        visited = set(nums)
        maxLength = 0

        for num in nums:
            if (num - 1) not in visited:
                cur_num = num
                length = 1

                while (cur_num + 1) in visited:
                    cur_num += 1
                    length += 1
                
                maxLength = max(maxLength, length)
    
        return maxLength
        